client/my-sites/themes/theme-showcase.jsx

```diff
@@ -626,12 +623,6 @@ class ThemeShowcase extends Component {
 					onPatternAssemblerButtonClick={ this.onDesignYourOwnClick }
 					isSiteWooExpressOrEcomFreeTrial={ isSiteWooExpressOrEcomFreeTrial }
 				/>
-				{ isLoggedIn && (
-					<ThemeShowcaseSurvey
```

Comment:
Since the component isn't used anywhere after this change, I'd remove it and bring it back again if the need arises. Wdyt?

Reply:
I don't have a strong opinion on this. 🤔 
The component is not imported anywhere anymore, so it doesn't affect anything.
Since we'll bring back the survey sooner rather than later, I reasoned that it's more convenient to just reimport it, rather than revert this PR.

I wonder if this PR should be even lighter, and just remove the survey prop instead of the whole logic, so that everything is already in place for the next survey.
Something like this:
```js
<ThemeShowcaseSurvey
  survey={ null }
  condition={ () => lastNonEditorRoute.includes( 'theme/' ) }
/>
```

Reply:
> The component is not imported anywhere anymore, so it doesn't affect anything.
Since we'll bring back the survey sooner rather than later, I reasoned that it's more convenient to just reimport it, rather than revert this PR.

👍

> I wonder if this PR should be even lighter, and just remove the survey prop instead of the whole logic, so that everything is already in place for the next survey.

I'm fine with both approaches.